components/form/index.zh-CN.md

```diff
@@ -115,14 +115,14 @@ const validateMessages = {
 表单字段组件，用于数据双向绑定、校验、布局等。
 
 | 参数 | 说明 | 类型 | 默认值 | 版本 |
-| --- | --- | --- | --- | --- |
+| --- | --- | --- | --- | --- | --- |
 | colon | 配合 `label` 属性使用，表示是否显示 `label` 后面的冒号 | boolean | true |  |
 | dependencies | 设置依赖字段，说明[见下](#dependencies) | [NamePath](#namepath)\[] | - |  |
 | extra | 额外的提示信息，和 `help` 类似，当需要错误信息和提示文案同时出现时，可以使用这个。 | ReactNode | - |  |
 | getValueFromEvent | 设置如何将 event 的值转换成字段值 | (..args: any\[]) => any | - |  |
 | getValueProps | 为子元素添加额外的属性 | (value: any) => any | - | 4.2.0 |
 | hasFeedback | 配合 `validateStatus` 属性使用，展示校验状态图标，建议只配合 Input 组件使用 | boolean | false |  |
-| customFeedbackIcons | 与“hasFeedback”一起使用，此选项将反馈图标替换为给定的图标。 | ({status:ValidateStatus, errors: ReactNode, warnings: ReactNode}) => Record<ValidateStatus,ReactNode> | - | 5.7.3 |
+| customFeedbackIcons | 与“hasFeedback”一起使用，此选项将反馈图标替换为给定的图标。 | ({status:ValidateStatus, errors: ReactNode, warnings: ReactNode}) => Record<ValidateStatus,ReactNode | boolean> | - | 5.7.3 |
```

Comment:
```
hasFeedback={{ icons }}
```

Reply:
do you want me to put the icons into the hasFeedback prop?

Reply:
> ```
> hasFeedback={{ icons }}
> ```

do you want me to put the icons into the hasFeedback prop?

Reply:
Yes, customFeedbackIcons depends on `hasFeedback`, it is better to put this into hasFeedback prop instead of creating a another property.

Reply:
> 

I see, thanks for the feedback. Already implemented as you told and requested a new review.